x-pack/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -32,7 +32,7 @@ function App() {
   useUpdateBadgeEffect();
 
   return (
-    <UrlParamsProvider>
+    <UrlParamsProvider history={history}>
```

Comment:
Any reason not to get `location` via the `useLocation` hook?

Reply:
It needs the whole `history` object, as it calls `history.replace()` (in `refreshTimeRange`).